helios-services/src/main/java/com/spotify/helios/master/MasterParser.java

```diff
@@ -79,9 +107,46 @@ protected void addArgs(final ArgumentParser parser) {
         .type(Integer.class)
         .setDefault(5802)
         .help("admin http port");
+
+    authenticationScheme = parser.addArgument("--auth-scheme")
+        .metavar("SCHEMENAME")
+        .help("Name of authentication scheme to use. "
+              + "Setting this flag enables authentication in Helios. "
+              + "'crtauth' support is built into Helios. "
```

Comment:
I don't think it is useful enough to mention here in the argparse help message, but I will add a note to authentication.md that it can be used (but really shouldn't, since it is so insecure) and which environment variables need to be set.